.circleci/config.yml

```diff
@@ -0,0 +1,188 @@
+          root: ~/
+          paths:
+            - workflow-start
+
+  ## Build jobs
+
+  ### build: build_{package}
+  ### Builds {package} and stores them in a cache.
+  build:
+    <<: *image
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/
+      - restore_cache: *yarn_cache
+      - restore_cache:
+          key: build_cache-v{{ .Environment.CI_CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}-{{ checksum "../workflow-start" }}-
+      - run: *extract_build_archive
+      - run:
+          name: Dependencies
+          command: sudo apt install -y rsync
+      - run: *git_submodule_install
+      - run: *yarn_install
+      - save_cache:
+          <<: *yarn_cache
+          paths:
+          - ~/.cache/yarn
+      - run:
+          name: Build
+          command: |
+            package="${CIRCLE_JOB#build_}"
+            package="${package//_/-}"
+
+            cd "packages/web-components/@sfx/${package}"
+            yarn build
+      - run: *archive_build
+      - save_cache:
+          key: build_cache-v{{ .Environment.CI_CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}-{{ checksum "../workflow-start" }}-{{ .Environment.CIRCLE_JOB }}-{{ epoch }}
+          paths:
+            - ~/build.tar
+
+  build_final:
+    <<: *image
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/
+      - restore_cache:
+          key: build_cache-v{{ .Environment.CI_CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}-{{ checksum "../workflow-start" }}-build_autocomplete-
+      - run: *extract_build_archive
+      - restore_cache:
+          key: build_cache-v{{ .Environment.CI_CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}-{{ checksum "../workflow-start" }}-build_base-
+      - run: *extract_build_archive
+      - restore_cache:
+          key: build_cache-v{{ .Environment.CI_CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}-{{ checksum "../workflow-start" }}-build_sayt-
+      - run: *extract_build_archive
+      - restore_cache:
+          key: build_cache-v{{ .Environment.CI_CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}-{{ checksum "../workflow-start" }}-build_ui-
+      - run: *extract_build_archive
+      - run: *archive_build
+      - save_cache:
+          key: build_cache-v{{ .Environment.CI_CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}-{{ checksum "../workflow-start" }}-all-{{ epoch }}
+          paths:
+            - ~/build.tar
+
+  ## Unit test jobs
+
+  ### unit_test: unit_test_{package}
+  ### Runs the tests for {package}.
+  unit_test:
+    <<: *image
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/
+      - restore_cache: *yarn_cache
+      - restore_cache: *build_final_cache_partial
+      - run: *extract_build_archive
+      - run: *yarn_install
+      - run:
+          name: Test
+          command: |
+            package="${CIRCLE_JOB#unit_test_}"
+            package="${package//_/-}"
+
+            cd "packages/web-components/@sfx/${package}"
```

Comment:
My assumption is that we'll have unique package names, so if the assumption that I made in the defaults section is true, we can change this to `cd packages/*/"@sfx/${package}"` so that when we add the wrappers, we don't have to remember to change this (or worse, not realize that we don't have to make a new job).